lib/pageflow/configuration.rb

```diff
@@ -266,6 +266,11 @@ class Configuration
     # @since edge
     attr_reader :permissions
 
+    # Defines the editor lock polling interval.
+    # @return [number]
+    # @since edge
+    attr_accessor :edit_lock
+
```

Comment:
Sorry for being so nitpicky, but when I suggested to rename `editor_lock_interval` and use `edit_lock` instead, I did not mean to drop the `interval` suffix. I'd even go for a more explicit naming like `edit_lock_polling_interval`.

Reply:
Oh sure, I'll changed it right now